staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -2181,6 +2187,19 @@ func describeJob(job *batchv1.Job, events *corev1.EventList) (string, error) {
 	})
 }
 
+func capIndexesList(indexes string, softLimit int) string {
+	ix := softLimit
+	for ; ix < len(indexes); ix++ {
+		if indexes[ix] == ',' {
+			break
+		}
+	}
+	if ix >= len(indexes) {
+		return indexes
+	}
+	return indexes[:ix+1] + "..."
```

Comment:
do we expect indexes to be completed in order? i.e. is index N likely to complete before index N+K for large K?

if so, since we're cutting the string off, the last 50 chars might be more informational to know how much is complete (although I'll concede that it'll be a little weird to read at first)

WDYT?

Reply:
if parallelism is smaller than completions, yes, smaller indexes will complete earlier.

What should happen is that the first interval should contain most of the values, like `0-123456`. The rest of the values would be more sporadic. I think the first interval actually gives you the most information: all indexes up to 123456 have completed, and there are some other completions here and there.